gcovr/formats/html/default/functions_page.content.html

```diff
@@ -1,36 +1,23 @@
 {# -*- engine: jinja -*- #}
 <table class="listOfFunctions">
   <tr>
-    <th>Function</th>
-    <th>File</th>
-    <th>Line</th>
+    <th>Function (File:Line)</th>
     <th>Call count</th>
     <th>Block coverage</th>
   </tr>
 {% for entry in all_functions %}
   <tr>
-    <td>
-      <a href="{{ entry['html_filename'] }}#l{{ entry['line'] }}">{{ entry["name"] }}</a>
-    </td>
-    <td>
-      <a href="{{ entry['html_filename'] }}#l{{ entry['line'] }}">{{ entry["filename"] }}</a>
-    </td>
-    <td>
-      <a href="{{ entry['html_filename'] }}#l{{ entry['line'] }}">{{ entry["line"] }}</a>
-    </td>
-    <td>
-      <a href="{{ entry['html_filename'] }}#l{{ entry['line'] }}">
-        {%- if entry["excluded"] %} excluded
-        {%- else %}
-        {%-  if entry["count"] == 0 %} not called
-        {%-  else %} called {{ entry["count"] }} time{% if entry["count"] > 1 %}s{% endif %}, returned {{ entry["returned"] }} time{% if entry["returned"] > 1 %}s{% endif %}
-        {%-  endif -%}
-        {%- endif -%}
-      </a>
-    </td>
-    <td>
-      <a href="#l{{ entry['line'] }}">{{ entry["blocks"] }}%</a>
-    </td>
-</tr>
+    {#- #}<td><a href="{{ entry['html_filename'] }}#l{{ entry['line'] }}">{{ entry["name"] }} ({{ entry["filename"] }}:{{ entry["line"] }})</a></td>
+    {#- #}<td>
+            {%- if entry["excluded"] %} excluded
```

Comment:
Suggest removing the whitespace before the word excluded.

Also, whitespace on lines 14 and 15.